src/Agent/NewRelic/Agent/Extensions/NewRelic.Agent.Extensions/Lambda/LambdaEventHelpers.cs

```diff
@@ -78,7 +75,6 @@ public static void AddEventTypeAttributes(IAgent agent, ITransaction transaction
                     transaction.AddEventSourceAttribute("length", (string)s3Event.Records.Count.ToString());
                     transaction.AddEventSourceAttribute("region", (string)s3Event.Records[0].AwsRegion);
                     transaction.AddEventSourceAttribute("eventName", (string)s3Event.Records[0].EventName);
-                    // TODO: Figure out if the eventTime value should be in some specific format. The spec doesn't say.
```

Comment:
I'm a bit surprised that these aren't called out anywhere. There's a big difference between UTC/local time, date localization, etc.